bundles/org.openhab.core.config.core/src/test/java/org/openhab/core/config/core/status/ConfigStatusInfoTest.java

```diff
@@ -90,17 +87,17 @@ public void assertCorrectConfigErrorHandlingUsingAddConfigError() {
         assertConfigStatusInfo(info);
     }
 
-    @Test
-    public void assertNPEisThrownIfTypesAreNull() {
-        ConfigStatusInfo info = new ConfigStatusInfo();
-        assertThrows(NullPointerException.class, () -> info.getConfigStatusMessages(null, emptySet()));
-    }
-
-    @Test
-    public void assertNPEisThrownIfParameterNamesAreNull() {
-        ConfigStatusInfo info = new ConfigStatusInfo();
-        assertThrows(NullPointerException.class, () -> info.getConfigStatusMessages(emptySet(), null));
-    }
+    // @Test
+    // public void assertNPEisThrownIfTypesAreNull() {
+    // ConfigStatusInfo info = new ConfigStatusInfo();
+    // assertThrows(NullPointerException.class, () -> info.getConfigStatusMessages(null, Set.of()));
+    // }
+    //
+    // @Test
+    // public void assertNPEisThrownIfParameterNamesAreNull() {
+    // ConfigStatusInfo info = new ConfigStatusInfo();
+    // assertThrows(NullPointerException.class, () -> info.getConfigStatusMessages(Set.of(), null));
+    // }
```

Comment:
I think such tests are quite useless when the classes are annotated.

```suggestion
```

Reply:
Sure. Done.